.github/workflows/ci.yaml

```diff
@@ -480,6 +479,40 @@ jobs:
         run: |
           poetry run pytest tests/ -vvv -n 2 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
 
+  update-docs-examples:
+    name: Update examples in doc schemas 
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    strategy:
+      fail-fast: true
+    env:
+      VALGRIND: 0
+      GENERATE_EXAMPLES: 1
+      PYTEST_OPTS: --timeout=1200
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          bash -x .github/scripts/setup.sh
+          pip install -U pip wheel poetry
+          poetry self add poetry-plugin-export
+          # Export and then use pip to install into the current env
+          poetry export -o /tmp/requirements.txt --without-hashes --with dev
+          pip install -r /tmp/requirements.txt
+      - name: Test Examples
+        env:
+          VALGRIND: 0
+          GENERATE_EXAMPLES: 1
+          PYTEST_OPTS: --timeout=1200      
+        run: |
+          ./configure --enable-debugbuild --enable-rust CC=clang
+          make -j $(nproc) check-doc-examples
```

Comment:
I think you want to unpack a previously-build version, e.g.:

```
      - name: Install bitcoind
        env:
          TEST_NETWORK: ${{ matrix.TEST_NETWORK }}
        run: .github/scripts/install-bitcoind.sh

      - name: Download build
        uses: actions/download-artifact@v4
        with:
          name: cln-${{ matrix.CFG }}.tar.bz2

      - name: Unpack pre-built CLN
        env:
          CFG: ${{ matrix.CFG }}
        run: |
          tar -xaf cln-${CFG}.tar.bz2
```

(probably CFG=compile-gcc ?)

Reply:
Thanks, it worked!!!